### DISCUSSION TITLE
Control Select Specific Slider Anchors

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#23266] [Ideas]
Thought this would be a cool feature to have, not sure about the demand but I've run into cases where this would be nice to have. It would be nice to be able to select specific anchor points on the slider (with CTRL) and be able to do actions with it, such as scale, rotate, or even flip them, leaving the other anchors fixed.

**Comment by peppy** [comment]
Can agree with this one. It looks like the selection part would be quite easy to implement locally in `PathControlPointVisualiser` (if we aren't interested in having it handled in line with the main hit object selection, which would be substantially larger change but may allow rotation/scale to work better?).

**Comment by Trung0246** [comment]
My alternative is having Control Point to be a separate object that show ups like notes and slider at the left selection bar. Then when you select both a slider and a control point, you can right click and select "Bind to selected slider". The binded control point will have the same original behavior, while the standalone control point will acts like a separate note with allowed action like rotate, scale, dupe, flip,.... Maybe these standalone control points can even show up on the top time line.
